pkg/gen/input/makefile/internal/file/makefile.go

```diff
@@ -39,6 +58,35 @@ build-darwin: $(APPLICATION)-darwin
 ## build-linux: builds a local binary for linux/amd64
 build-linux: $(APPLICATION)-linux
 
+{{- if eq .CurrentFlavour .FlavourCLI}}
+
+.PHONY: package-darwin package-linux
+## package-darwin: prepares a packaged darwin/amd64 version
+package-darwin: $(APPLICATION)-package-darwin
+## package-linux: prepares a packaged linux/amd64 version
+package-linux: $(APPLICATION)-package-linux
+
+$(APPLICATION)-package-%: $(SOURCES)
+	docker run --rm \
+    		-v $(shell pwd):/$(APPLICATION) \
+    		-w /$(APPLICATION) \
+    		-e GOOS=$* -e GOARCH=amd64 \
+    		golang:$(GOVERSION)-alpine go build \
+    		-ldflags "$(LDFLAGS)" \
+    		-o $(APPLICATION)-$(VERSION)-$*-amd64
+	@$(MAKE) $(APPLICATION)-archive-$*
+
+$(APPLICATION)-archive-%:
```

Comment:
```suggestion
$(APPLICATION)-package-%: $(APPLICATION)-%
```

If the motivation here is to enforce certain version of Go in CI then we should make sure the desired version of Go is installed in the CI environment. In CircleCI you'd you [architect container like here](https://github.com/giantswarm/apiextensions/blob/2c983ba811b7917e0025533132e17ff0ee04de6a/.circleci/config.yml#L7-L17). In GH actions you can use container as well but I think there is an easy way to enforce certain version of Go in VM.

To me it's very important to have `go build` defined in a single place.

Reply:
In GH actions it should be a matter fo setting `GOROOT` to `GOROOT_1_14_X64`. See https://github.com/actions/virtual-environments/blob/6f246a3b4fd9824137a586543dc8aeec3b7f35a6/images/linux/scripts/installers/go.sh#L20

Reply:
Yeah, that was the reason for using a container.

I changed it up, it's now building using the generic configuration. I'll handle explicit go versions in the GitHub action.

Thank you 👊